docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -93,21 +94,30 @@ else:
 
 **Inputs**:
 
-*   **1**: `data` -- input tensor of rank 3 or greater. Layout is `[N, C_INPUT, X1, ..., XD]`. *Required*.
+*   **1**: Input tensor of type *T1* and rank 3, 4 or 5. Layout is NCZYX (number of batches, number of channels, spatial axes Z, Y, X). *Required*.
 
-*   **2**: `filter` -- convolution kernel tensor. Weights have shape `[C_INPUT, C_OUTPUT, K_D, ..., K_1]`. `C_INPUT` is the number of channels in input `data` tensor shape, and `C_OUTPUT` is the number of channels in the `output` tensor. Spatial size of the kernel `[K_D, ..., K_1]` is derived from the shape of this input and aren't specified by any attribute. *Required*.
+*   **2**: Convolution kernel tensor of type *T1* and rank 3, 4 or 5. Layout is IOZYX (number of input channels, number of output channels, spatial axes Z, Y, X). Spatial size of the kernel is derived from the shape of this input and aren't specified by any attribute. *Required*.
```

Comment:
Removing shape layout may mislead the user. It looks like with respect to Convolution, the dimensions are reversed.
filters in Convolution have shape `[C_OUT, C_IN, Z, Y, X]`, while in Backprop the shape is `[C_INPUT, C_OUTPUT, K_D, ..., K_1]`. Also based on current reference implementation for ConvBackprop ([here](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/reference/include/ngraph/runtime/reference/convolution_backprop_data.hpp#L252)) I see that spatial dimensions of data input are reversed. @lazarevevgeny maybe you could give a comment here?

Reply:
@dmitry-gorokhov could you explain how it works from CPU plugin point of view?

Reply:
@sdurawa As you correctly mentioned CPU plug-in expects weights in [C_INPUT, C_OUTPUT, K_D, ..., K_1] layout.

Reply:
Filter layout changed to the same as in cpu plugin.